src/Driver/PDO/Connection.php

```diff
@@ -78,9 +78,9 @@ public function query(string $sql): ResultInterface
     /**
      * {@inheritdoc}
      */
-    public function quote($value, $type = ParameterType::STRING)
+    public function quote($value, int $type = ParameterType::STRING)
     {
-        return $this->connection->quote($value, $type);
+        return $this->connection->quote($value, Statement::convertParamType($type));
```

Comment:
Why is this change of behavior suddenly necessary? If this is a bugfix, please provide a functional test that fails without this change.